### PR TITLE
Add automated "outstanding skill" notification. Closes #10.

### LINKED ISSUE #10: add automated "outstanding expertise" notification
Todos:
- [ ] Port it over!

Questions:
- [ ] should this be scheduled via JS or cron?
- [ ] can this be written so that it can be triggered manually?

### PR DESCRIPTION
**Pull request by cowboy**
Notes:
- `RUN_JOBS=1` has been added to `/mnt/secrets/skillsbot/slack-production` so this will need to be run before deploying:
  - `./deploy/run-playbook.sh configure production --tags=services`
- The `job missing` hidden command will run the job manually, for debugging purposes, but the output will be sent to you instead of to the users.

### CODE REVIEW

**Comment by cowboy** [comment]
@tkellen can I get a review? Especially of my SQL!

You should be able to test with `/dm @skillsbot_cowboy job missing`

File: src/sql/outstanding_skills_for_team.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+SELECT
```
Comment (tkellen) [inline src/sql/outstanding_skills_for_team.sql:1]: :+1:

**Comment by tkellen** [comment]
SQL looks sensible to me! I like that you can demo a run for yourself. Any reason not to make it possible to kick one off for real too? Once the admin commands are in place I could see someone on the sales team looking for quick feedback wanting to do that, for example.

**Comment by cowboy** [comment]
I'm not sure I like the idea of triggering a mass "outstanding skills" message manually. Here are a few reasons:
- The same message is already sent out daily...
- Which means that either the mass "outstanding skills" notification hasn't been sent out today, in which case it'll be sent out soon, or the message has already been sent out today, in which case it'll be sent out tomorrow at noon (or earlier, depending on your time zone).
- Users might feel annoyed at receiving 2 messages from the bot in one day.
- Just because a message is sent out to everyone doesn't mean everyone will actually update their skills immediately.

Is it expected to be common that someone will want to add a new skill and need experience and interest levels from everyone for that skill before noon, the next day?

**Comment by tkellen** [comment]
To your specific question (RE: it being common that someone will want to add a new skill), I don't think it has a chance of becoming common until people start using the system more. 

To the rest of your objections--I think those are fair. I'll file an issue (probably should have done that anyway). I don't think my idea should hold this PR up.

**Pull request merged**